src/ai/backend/web/config.py

```diff
@@ -75,6 +75,7 @@
                 t.Key("directory_based_usage", default=False): t.ToBool(),
                 t.Key("allow_custom_resource_allocation", default=True): t.ToBool(),
                 t.Key("edu_appname_prefix", default=""): t.String(allow_blank=True),
+                t.Key("support_model_store", default=True): t.ToBool(),
```

Comment:
The name should be `enable_model_store`.
"Not support" means that we don't have the ability to serve the model store.
Here we should be clear that we DO support it but it's simply disabled when this field is set false.